packages/crypto/src/IncrementalMerkleTree.ts

```diff
@@ -10,7 +16,9 @@ export class IncrementalMerkleTree extends zkIncrementalMerkleTree {
      * @param zeroValue Zero values for zeroes.
      * @param arity The number of children for each node.
      */
-    constructor(depth: number, zeroValue: any, arity?: number) {
```

Comment:
This export is a a monkey patch on top of `zk-kit`. Instead of doing this I think it would be better to remove this export entirely and have downstream code depend directly on `zk-kit`.

Reply:
yeah crypto doesn't have to maintain this. Just to make downsteam easier to use